src/renderer/themes.css

```diff
@@ -1896,6 +2096,190 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --accent-color-opacity4: rgb(187 188 242 / 24%);
 }
 
+.secEverforestDarkRed,
+.secEverforestDarkOrange,
+.secEverforestDarkYellow,
+.secEverforestDarkGreen,
+.secEverforestDarkAqua,
+.secEverforestDarkBlue,
+.secEverforestDarkPurple,
+.secEverforestLightRed,
+.secEverforestLightOrange,
+.secEverforestLightYellow,
+.secEverforestLightGreen,
+.secEverforestLightAqua,
+.secEverforestLightBlue,
+.secEverforestLightPurple {
+  --text-with-accent-color: #1e2326;
+}
```

Comment:
**issue (blocking):** You have a block I was requesting above for the secondary theme colors, but this white text color is too bright for some of the light themes. See above comment; you can copy most of the work for the `main`s to the `sec`s.

![Screenshot_20241230_151455](https://github.com/user-attachments/assets/1db4ba38-69bd-4721-bc72-8b4aaa346f3c)